haiku-apps/xrs/xrs-1.9.1.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station.
+You can use it to create music compositions and loops of your preferred elettronic music genres."
+
+HOMEPAGE="https://github.com/Freaxed/xrs-haiku"
+COPYRIGHT="2006-2022, Andrea Anzani.
+	2017-2020, Andrew Lindesay.
+	2012, Gerasim Troeglazov (3dEyes**).
+	2001-2010 Stephan Aßmus.
+	2006 Javier Serrano Polo.
+	1999 Fabien Fulhaber.
+	1998-1999 Be Incorporated.
+	"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Freaxed/xrs-haiku/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="c2b89a858b4e0ff29104462f7cab80ce318d3c730b559923f5f2477ea687103e"
+SOURCE_DIR="xrs-haiku-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	xrs$secondaryArchSuffix = $portVersion
+	app:XRS = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfluidlite$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libfluidlite$secondaryArchSuffix
+	devel:libsamplerate$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	make
```

Comment:
```suggestion
	make $jobArgs
```

this can speed up the build, can break the build if dependencies aren't correct defined.